src/mscorlib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -138,10 +138,10 @@ public AdjustmentRule[] GetAdjustmentRules()
             {
                 var rule = _adjustmentRules[i];
                 var start = rule.DateStart.Kind == DateTimeKind.Utc ?
-                            new DateTime(TimeZoneInfo.ConvertTime(rule.DateStart, this).Ticks, DateTimeKind.Unspecified) :
+                            new DateTime(rule.DateStart.Ticks + _baseUtcOffset.Ticks, DateTimeKind.Unspecified) :
```

Comment:
Is it worth a comment explaining why one has the daylightdelta and not the other? After all, that confused whoever wrote the original code.